prep_data_school.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+import os.path
+import sys
+from pathlib import PurePath, PurePosixPath
+from posixpath import abspath, dirname
+from typing import List, Optional
+
+import pandas as pd
+import unidecode
+from numpy import dtype, str0
+
+
+def remove_characters(input_text: str, chars_to_remove: List[str]) -> str:
+    """Method for data preparation
+
+    Parameters
+    ----------
+    input_text from df
+    chars_to_remove in input_text
+    """
+
+    for char_to_remove in chars_to_remove:
+        input_text = input_text.replace(char_to_remove, "")
+    return input_text.strip()
+
+
+def prep_data(path_input: str, path_output: Optional[str] = None):
+    """Method for data preparation
+
+    Parameters
+    ----------
+    path_input
+        Input csv path
+    path_output
+        Optional output csv path
+    """
+    file_input = "school.csv"
+    file_output = "school_prep.csv"
+    # assert os.path.exists(path_input), "This file does not exist!"
+
+    if not os.path.exists(path_input):
+        # raise FileExistsError
+        print("Sciezka nie istnieje!")
```

Comment:
Jeśli nie istnieje to przerywamy, żeby się dalej nie wykonywał bo i tak się wywali
```suggestion
        print("Sciezka nie istnieje!")
        return
```

Reply:
ok